src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -2077,6 +2077,11 @@ private void doDeleteServiceInstance(CloudServiceInstance serviceInstance) {
 
     private void doUnbindServiceInstance(UUID applicationGuid, UUID serviceInstanceGuid) {
         UUID serviceBindingGuid = getServiceBindingGuid(applicationGuid, serviceInstanceGuid);
+        if (serviceBindingGuid == null) {
+            throw new CloudOperationException(HttpStatus.NOT_FOUND,
+                                              "Not Found",
+                                              "Not found service binding between service with GUID " + serviceInstanceGuid + " and application with GUID " + applicationGuid + ".");
```

Comment:
Maybe move the "Not found" to the end of the sentence, so it's `Service binding between service with GUID ... and application with GUID ... not found.`

Reply:
+1

Reply:
Done